tests/io/test_data_catalog.py

```diff
@@ -673,8 +675,10 @@ def test_replacing_nonword_characters(self):
         assert "ds3__csv" in catalog.datasets.__dict__
         assert "jalapeño" in catalog.datasets.__dict__
 
-    def test_no_versions_with_cloud_protocol(self):
+    def test_no_versions_with_cloud_protocol(self, monkeypatch):
         """Check the error if no versions are available for load from cloud storage"""
+        monkeypatch.setenv("AWS_ACCESS_KEY_ID", "dummmy")
+        monkeypatch.setenv("AWS_SECRET_ACCESS_KEY", "dummmy")
         version = Version(load=None, save=None)
         versioned_dataset = CSVDataSet("s3://bucket/file.csv", version=version)
         pattern = re.escape(
```

Comment:
It's only needed for Python 3.11, maybe not the best fix but I cannot find another way. If I don't do this, I will get `DataSetError: Unable to locate Credentials` instead.